openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum.py

```diff
@@ -101,56 +61,50 @@ def _handle_response(batch):
         else:
             width, height = None, None
 
-        title = data.get("title", None)
-        metadata = _get_metadata(data)
-        if data.get("creators"):
-            creator_name = data.get("creators")[0].get("description", "")
+        title = record.get("title", None)
+        metadata = self._get_metadata(record)
+        if record.get("creators"):
+            creator_name = record.get("creators")[0].get("description", "")
         else:
             creator_name = ""
         license_info = get_license_info(
             license_=license_, license_version=license_version
         )
-        total_images = image_store.add_item(
-            foreign_landing_url=foreign_landing_url,
-            image_url=image_url,
-            license_info=license_info,
-            foreign_identifier=foreign_id,
-            width=width,
-            height=height,
-            title=title,
-            creator=creator_name,
-            meta_data=metadata,
-        )
-    return total_images
-
-
-def _get_image_type(image_data):
-    key, image_url = None, None
-    if image_data.get("web"):
-        key = "web"
-        image_url = image_data.get("web").get("url", None)
-    elif image_data.get("print"):
-        key = "print"
-        image_url = image_data.get("print").get("url", None)
-    elif image_data.get("full"):
-        key = "full"
-        image_url = image_data.get("full").get("url", None)
-    return image_url, key
-
-
-def _get_metadata(data):
-    metadata = {
-        "accession_number": data.get("accession_number", ""),
-        "technique": data.get("technique", ""),
-        "date": data.get("creation_date", ""),
-        "credit_line": data.get("creditline", ""),
-        "classification": data.get("type", ""),
-        "tombstone": data.get("tombstone", ""),
-        "culture": ",".join([i for i in data.get("culture", []) if i is not None]),
-    }
-    metadata = {k: v for k, v in metadata.items() if v is not None}
-    return metadata
-
 
-if __name__ == "__main__":
-    main()
+        return {
+            "foreign_landing_url": foreign_landing_url,
+            "image_url": image_url,
+            "license_info": license_info,
+            "foreign_identifier": foreign_id,
+            "width": width,
+            "height": height,
+            "title": title,
+            "creator": creator_name,
+            "meta_data": metadata,
+        }
+
+    def _get_image_type(self, image_data):
```

Comment:
Pycharm is complaining that this method (and the `_get_metadata` below it) can be `static` because it doesn't use `self`. But I'm not sure what difference it makes.

Reply:
Good find! It seems like it's [generally good practice to use static methods where possible](https://stackoverflow.com/a/22589883/3277713), though it really wouldn't matter too much for this case since we're usually only instantiating one instance anyway.